why-graphql-advantages-disadvantages-alternatives.md

```diff
@@ -117,80 +117,72 @@ Facebook has been using GraphQL since 2012, so even before it got open sourced.
 
 {{% sub_chapter_header "GraphQL Rate Limiting" "graphql-rate-limiting" %}}
 
 Another problem is rate limiting. Whereas in REST it is simpler to say "we allow only so many resource requests in one day", it becomes difficult to make such a statement for individual GraphQL operations, because it can be everything between a cheap or expensive operation. That's where companies with {{% a_blank "public GraphQL APIs come up with their specific rate limiting calculations" "https://developer.github.com/v4/guides/resource-limitations/" %}} which often boil down to the previously mentioned maximum query depths and query complexity weighting.
 
 {{% sub_chapter_header "GraphQL Caching" "graphql-caching" %}}
 
-Implementing a simplified cache when having GraphQL becomes far more complex than implementing it in REST. In REST you access resources with URLs and thus you would be able to cache on a resource level, because you have the resource URL as identifier. In GraphQL this becomes complex, because each query can be different even though it operates on the same entity. In one query you may only request the name of an author but in the next query you want to know the email address too. That's where you need to have a more fine-grained cache in place on a field level which isn't the simplest thing to implement. However, most of the libraries build on top of GraphQL are offering these caching mechanisms out of the box for you.
+Implementing a simplified cache with GraphQL is more complex than implementing it in REST. In REST, resources are accessed with URLs,  you would be able to cache on a resource level, because you have the resource URL as identifier. In GraphQL, this becomes complex because each query can be different, even though it operates on the same entity. You may only request just the name of an author in one query, but want to know the email address in the next. That's where you need a more fine-grained cache at field level, which can be difficult to implement. However, most of the libraries built on top of GraphQL offer caching mechanisms out of the box.
```

Comment:
"In REST, resources are accessed with URLs,  you would" -> "In REST, resources are accessed with URLs. You would"